src/services/projects.js

```diff
@@ -5,8 +5,123 @@ const common = require('@constants/common')
 const filesService = require('@services/files')
 const userRequests = require('@requests/user')
 const _ = require('lodash')
+const fs = require('fs')
+const axios = require('axios')
+const FormData = require('form-data')
+const moment = require('moment-timezone')
+const path = require('path')
+const ROOT_PATH = path.join(__dirname, '..')
 
 module.exports = class ProjectsHelper {
+	/**
+	 *  project create
+	 * @method
+	 * @name create
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - project id
+	 */
+	static async create(orgId, loggedInUserId) {
+		try {
+			let projectData = {
+				type: common.PROJECT,
+				status: common.STATUS_DRAFT,
+				user_id: loggedInUserId,
+				organization_id: orgId,
+				meta: {},
+				created_by: loggedInUserId,
+				updated_by: loggedInUserId,
+			}
+			let projectCreate = await resourceQueries.create(projectData)
+			return responses.successResponse({
+				statusCode: httpStatusCode.ok,
+				message: 'PROJECT_CREATED_SUCCESSFULLY',
+				result: { id: projectCreate.id },
+			})
+		} catch (error) {
+			console.log(error, 'error')
+			throw error
+		}
+	}
+	/**
+	 * project update
+	 * @method
+	 * @name update
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - project update response.
+	 */
+
+	static async update(resourceId, orgId, loggedInUserId, bodyData) {
+		try {
+			let { categories, recommeneded_for, languages, ...projectData } = bodyData
+			categories = categories.map((key) => {
+				return { label: key, value: key }
+			})
+			recommeneded_for = recommeneded_for.map((key) => {
+				return { label: key, value: key }
+			})
+			languages = languages.map((key) => {
+				return { label: key, value: key }
+			})
+			projectData.categories = categories
+			projectData.languages = languages
+			projectData.recommeneded_for = recommeneded_for
+			let currentDate = moment(new Date())
+			let formattedDate = currentDate.format('DD-MM-YYYY')
+			if (!fs.existsSync(`${ROOT_PATH}/public/assets/${formattedDate}`)) {
+				fs.mkdirSync(`${ROOT_PATH}/public/assets/${formattedDate}`)
+			}
+			let localPath = `${ROOT_PATH}/public/assets/${formattedDate}/`
```

Comment:
@ankit1shahu you dont need to add as a file. You can directly add the json in the req.body, methods will be put and content type multipart/form-data